scripts/sanitycheck

```diff
@@ -1236,6 +1238,7 @@ class MakeGenerator:
             handler.pid_fn = os.path.join(instance.outdir, "renode.pid")
             handler.call_make_run = True
 
+        verbose("Test handler %s for %s" % (type, name))
```

Comment:
```
Test handler native for native_posix/samples/hello_world/sample.helloworld
Test handler None for em_starterkit_em7d/samples/hello_world/sample.helloworld
Test handler None for frdm_k64f/samples/hello_world/sample.helloworld
Spawning QEMU process for mps2_an385/samples/hello_world/sample.helloworld
Test handler qemu for mps2_an385/samples/hello_world/sample.helloworld
Test handler None for nrf51_pca10028/samples/hello_world/sample.helloworld
Spawning QEMU process for qemu_nios2/samples/hello_world/sample.helloworld
```

`Test handler None for...` you probably need to customise the message a bit for no handler.

Reply:
how about this?

```
 42/153 hifive1                   samples/hello_world/sample.helloworld              PASSED (build)
 43/153 lpcxpresso54114_m0        samples/hello_world/sample.helloworld              PASSED (build)
 44/153 native_posix              samples/hello_world/sample.helloworld              PASSED (native)
 45/153 lpcxpresso54114_m4        samples/hello_world/sample.helloworld              PASSED (build)
 46/153 iotdk                     samples/hello_world/sample.helloworld              PASSED (build)
 47/153 nrf51_pca10028            samples/hello_world/sample.helloworld              PASSED (build)
 48/153 mps2_an385                samples/hello_world/sample.helloworld              PASSED (qemu)
 49/153 mec2016evb_assy6797       samples/hello_world/sample.helloworld              PASSED (build)
 50/153 nrf51_blenano             samples/hello_world/sample.helloworld              PASSED (build)
 51/153 nrf51_vbluno51            samples/hello_world/sample.helloworld              PASSED (build)
 52/153 mimxrt1020_evk            samples/hello_world/sample.helloworld              PASSED (build)
 53/153 msp_exp432p401r_launchxl  samples/hello_world/sample.helloworld              PASSED (build)
 54/153 nrf52840_pca10059         samples/hello_world/sample.helloworld              PASSED (build)
 55/153 nrf52_sparkfun            samples/hello_world/sample.helloworld              PASSED (build)
 56/153 minnowboard               samples/hello_world/sample.helloworld              PASSED (build)
 57/153 nrf52_adafruit_feather    samples/hello_world/sample.helloworld              PASSED (build)
 58/153 m2gl025_miv               samples/hello_world/sample.helloworld              PASSED (renode)
 59/153 hexiwear_kw40z            samples/hello_world/sample.helloworld              PASSED (build)
```

I have a patch ready, will come shortly.

Reply:
Yes, much better than documenting the issue! I volunteered to do something like it on Slack before this but I was afraid to break backward compatibility with some screen-scraping script(s) and at the time you said you were not sure about how to do it.

Reply:
> Test handler None for... you probably need to customise the message a bit for no handler.

I liked it and it kept the code shorter.

> 56/153 minnowboard               samples/hello_world/sample.helloworld              PASSED (build)
 
While fixing the test output is of course massively better than explaining the test output in the documentation, I thought the latter had more chance to make it for the Long Term Support which is happening now with neither.

Reply:
> I thought the latter had more chance to make it for the Long Term Support which is happening now with neither.

we can backport all of this to 1.14.x branch, we just do not want to introduce any any changes right now couple of day before the release...

Reply:
@nashif submitted PR #15523